components/raftstore/src/coprocessor/split_check/size.rs

```diff
@@ -266,26 +266,21 @@ fn get_approximate_split_keys(
     max_size: u64,
     batch_split_limit: u64,
 ) -> Result<Vec<Vec<u8>>> {
-    let get_cf_size = |cf: &str| get_region_approximate_size_cf(db, cf, &region, 0);
-
-    let default_cf_size = box_try!(get_cf_size(CF_DEFAULT));
-    let write_cf_size = box_try!(get_cf_size(CF_WRITE));
-    if default_cf_size + write_cf_size == 0 {
-        return Err(box_err!("default cf and write cf is empty"));
+    let get_cf_size = |cf: &str| get_region_approximate_size_cf(db, cf, &region, 0).unwrap_or(0);
+    let cfs = [
+        (CF_DEFAULT, get_cf_size(CF_DEFAULT)),
+        (CF_WRITE, get_cf_size(CF_WRITE)),
+        (CF_LOCK, get_cf_size(CF_LOCK)),
+    ];
```

Comment:
I suggest:
```suggestion
    let get_cf_size = |cf: &str| get_region_approximate_size_cf(db, cf, &region, 0);
    let cfs = [
        (CF_DEFAULT, box_try!(get_cf_size(CF_DEFAULT))),
        (CF_WRITE, box_try!(get_cf_size(CF_WRITE))),
        (CF_LOCK, get_cf_size(CF_LOCK).unwrap_or(0)),
    ];
```
If something is wrong, the total_size will be zero forever.. I prefer not to swallow the error of CF_WRITE and CF_LOCK.